baseband/vlbi_base/payload.py

```diff
@@ -139,11 +139,33 @@ def dtype(self):
         return np.dtype(np.complex64 if self.complex_data else np.float32)
 
     def _item_to_slices(self, item):
-        """Get word and data slices required to get given item.
+        """Get word and data slices required to retrieve sample(s)
+        specified by ``item``.
 
-        Returns ``words_slice`` and ``data_slice`` such that if one decodes
-        ``self.words[words_slice]`` the returned data is the smallest possible
-        array that includes the requested item or slice (as ``data_slice``).
+        Parameters
+        ----------
+        item : int, slice, or tuple
+            Sample indices.  Int represents a single sample, slice
+            a sample range, and tuple of ints/slices a range for
+            multi-channel data.
+
+        Returns
+        -------
+        words_slice : slice
+            Slice such that if one decodes ``ds = self.words[words_slice]``,
+            ``ds`` is the smallest possible array that includes all
+            of the requested ``item``.
+        data_slice : int or slice
+            Int or slice such that ``ds[data_slice]`` is the requested
+            ``item``.
+
+        Notes
+        -----
+        Since ``item`` is restricted to (tuples of) ints or slices, cannot
```

Comment:
Grammar odd; maybe just delete `Since` and add `so one` after the comma.  Similarly, below `one cannot`. This part generally reads a bit oddly, maybe try to rewrite more as text, less telegramme style?